### DISCUSSION TITLE
Top learning resources for lfe / web framework?

### DISCUSSION THREAD

**Discussion by jgarte** [lfe/lfe#458] [Q&A]
@oubiwann 

What do you think are the top learning resources for learning `lfe` currently?

If I wanted to make a web app, what is the recommended web framework for lfe (something that wraps cowboy?)?

**Comment by oubiwann** [comment]
Oh, gosh ... December! Soooo sorry -- no idea how I missed this. 

There's no "official" web framework for LFE. Lots of folks love Cowboy (which is a great project); I have been a user of YAWS since 2007; I'm also a pretty big fan of Elli.

It's wild west, which means you can do whatever you want! 😉 (if you're still talking to us after being so badly abused by this response delay 😭). It's super-easy to translate from Erlang web apps to LFE (since LFE is essentially just a library on Erlang/OTP. If you have any questions/problems with translation, feel free to ping us here again, or on Slack/Discord ...
